docs/README.md

```diff
@@ -0,0 +1,19 @@
+## Building the documentation
+
+The documentation HTML is produced with the Ruby-based `jekyll` tool.
+
+1. Make sure Ruby 2.x is installed.
+2. `gem install jekyll`
+
+## Viewing the documentation locally
+
+Then to view the documentation in your local checkout:
+
+1. Before you begin, cd into `docs/` directory, and `cp _config.yml _config.local.yml`. Then edit `_config.local.yml` and change the `url:` value to `http://localhost:4000`. This local config file will be ignored by git. 
+1. In a separate shell session, `cd` to the `docs/` directory, and do:
+
+```jekyll serve --incremental --config _config.local.yml```
```

Comment:
You need to add line-breaks after the opening backticks and before the closing ones so that this lines turns into a code block instead.

```
(backticks)
jekyll serve --incremental --config _config.local.yml
(closing backticks)
```